src/nvme/ioctl.h

```diff
@@ -428,8 +432,6 @@ static int nvme_identify_cns_nsid(int fd, enum nvme_identify_cns cns,
 		.uuidx = NVME_UUID_NONE,
 		.csi = NVME_CSI_NVM,
 		.data = data,
-		.timeout = NVME_DEFAULT_IOCTL_TIMEOUT,
-		.result = NULL,
```

Comment:
And it's quetionable whether we need to move the initialisation around. Yes, doing so will align with the structure layout, but then we already have a named initializer, so we don't need to rely on the order anymore.
Which was kinda the point of the initial patchset to have a struct as argument.

Reply:
The idea here is that we have a all common members at the beginning. I was playing with the idea to have a default macro to set these, something like

```
struct nvme_foo_args args = {
    nvme_args_defaults(args, fd)
    .nsid   = 0x1,
    [...],
};
```

Not sure about that though.